### PR TITLE
Windows icon overlays

### PR DESCRIPTION
**Pull request by sarat-ravi**
We now have windows icon overlays!!

### CODE REVIEW

File: build.user.properties.template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+nativity.dir=C:/Users/sync/Dev/streem/liferay-nativity
```
Comment (tluthra) [inline build.user.properties.template]: is the only way to have this entire path here? we can't do a thing relative to user, or have this configurable?

File: build.user.properties.template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+nativity.version=1.0.2
+ms.sdk.7.1.dir=C:/Program Files/Microsoft SDKs/Windows/v7.1
+framework.dir=C:/Windows/Microsoft.NET/Framework64/v4.0.30319
+
+context.menu.guid={EBEDF891-A73A-4B35-91BB-57E3820C4BBC}
+
+overlay.name.synced=BoxDesktopSyncedOverlay
+overlay.guid.synced={1DEAD8A6-8293-411D-8201-C32E2D6F5005}
+overlay.id.synced=1
+overlay.path.synced=C:/Users/sync/Dev/streem/app/assets/overlays/Synced.ico
```
Comment (tluthra) [inline build.user.properties.template]: See other PR, but I don't think this icon should be called "synced"

File: java/src/com/liferay/nativity/Constants.java [hunk 3]
```diff
@@ -45,7 +45,7 @@
 	public static final String MENU_TEXT = "menuText";
 
 	public static final String NATIVITY_REGISTRY_KEY =
-		"SOFTWARE\\Liferay Inc\\Liferay Nativity";
+		"SOFTWARE\\Box Inc\\Box Desktop";
```
Comment (tluthra) [inline java/src/com/liferay/nativity/Constants.java:48]: Again, this was changed in the other PR, I feel like i'm re-reading a lot of stuff.  Did you mean to merge this with master? or the other branch?

File: build.user.properties.template (hunk 1 above)
Comment (sarat-ravi) [inline build.user.properties.template]: These paths are used in the liferay solution so if the paths are relative, they have to be relative to whatever the working directory is where the build is run. What we can do is have a script that generates a properties file for you, but that is not the scope of this PR. Besides, this is what liferay normally makes you do anyways

File: build.user.properties.template (hunk 2 above)
Comment (sarat-ravi) [inline build.user.properties.template]: According to our discussion, all the icons will be renamed soon. I will keep you posted about what they will be called in the future.

File: java/src/com/liferay/nativity/Constants.java (hunk 3 above)
Comment (sarat-ravi) [inline java/src/com/liferay/nativity/Constants.java:48]: Sorry, I didn't mean to open this PR before the other PR was merged in. I'll make sure I will maintain ordering/chaining properly next time

File: build.user.properties.template (hunk 1 above)
Comment (tluthra) [inline build.user.properties.template]: Yea, but this is now the first place in the codebase that requires a specific username, which is in my opinion way to strict of a restriction.  What happens when we want to build this in different VMs, or on Jenkins machines that don't have the username 'sync'?

Comment (sarat-ravi) [inline build.user.properties.template]: Ugh yeah, we need to have a generator script that creates the appropriate properties file then. I didn't think about the jenkins problem.

Comment (sarat-ravi) [inline build.user.properties.template]: Changed ant script/installer scripts to just work, regardless of the username. Also, made it more jenkins friendly.

**Comment by tluthra** [comment]
:+1:

**Comment by nealwu** [comment]
:+1:

**Pull request merged**